scripts/embedwast.py

```diff
@@ -1,31 +1,45 @@
-#! /usr/bin/env python
-
-#   Copyright 2018 WebAssembly Community Group participants
+#!/usr/bin/env python
+#
+# Copyright 2018 WebAssembly Community Group participants
 #
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-#       http://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
 #
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 import sys
 
-wast = open('wasm-intrinsics.wast').read()
+input_file = sys.argv[1]
+output_file = sys.argv[2]
+
+with open(input_file) as f:
+  wast = f.read()
+
+output = """\
+// Automatically generated by embedwast.py
+
+#include "passes/intrinsics-module.h"
+
+static const char theModule[%d] = {
+""" % (len(wast) + 1)
 
-output = ""
-output += "#include \"passes/intrinsics-module.h\"\n"
-output += "namespace wasm {\n"
-output += "const char theModule[" + str(len(wast) + 1) + "] = {\n"
 for c in wast:
-    output += str(ord(c)) + ', '
-output += '0'
-output += "\n};\n"
-output += "const char* IntrinsicsModuleWast = theModule;\n"
-output += "}\n"
-open(sys.argv[1], 'w').write(output)
+  output += str(ord(c)) + ', '
+
+output += '''0
```

Comment:
nit: why is this triple-quoted string using `'''`, when the other one uses `"""`?
Is there a standard for which to use? Either way these should be the same.

Reply:
Personally I think `'` is nicer as it requires fewer keypresses on most keyboards.